app/jenkins-for-jira-ui/src/components/InProductHelpDrawer/InProductHelp.styles.tsx

```diff
@@ -7,10 +7,14 @@ export const inProductHelpActionLink = css`
 	color: ${token('color.link')};
 	margin-bottom: ${token('space.025')};
 	padding: ${token('space.0')} !important;
+	white-space: normal !important;
 
 	span {
     font-weight: normal;
     margin: ${token('space.0')};
+		line-height: 20px;
+		text-align: left;
+		white-space: normal !important;
```

Comment:
Needed to add this in as it turns out really long sentences become 'This is a really long sentence...' and there's no way to wrap them.

Reply:
Is there a way to target this element in a more specific manner so that it overrides? Using ids or classNames?